src/entry.js

```diff
@@ -0,0 +1,18 @@
+const fromPairs = require('lodash/fromPairs');
+const assign = require('lodash/assign');
+
+const mergeAll = (accum, current) => assign(accum, current);
+
+const contexts = [
+  require.context('./atoms', true, /.*\.js/),
+  require.context('./molecules', true, /.*\.js/),
+  require.context('./organisms', true, /.*\.js/),
+  require.context('./templates', true, /.*\.js/),
+  // require.context('./pages', true, /.*\.js/),
```

Comment:
@j-clark I'm assuming that this is here just in case we want to add it later? If so, maybe add a little comment or even remove it, since it's highly unlikely that we'll need to export whole compositions of PG page examples.